core/templates/components/skill-mastery/skill-mastery.component.ts

```diff
@@ -0,0 +1,84 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the skill mastery viewer.
+ */
+
+import { SkillMasteryListConstants } from
+  'components/skills-mastery-list/skills-mastery-list.constants';
+
+import { Component, Input, OnInit } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+
+import { SkillMasteryBackendApiService } from
+  'domain/skill/skill-mastery-backend-api.service';
+
+@Component({
+  selector: 'skill-mastery-viewer',
+  templateUrl: './skill-mastery.directive.html',
```

Comment:
Rename the html file to `skill-mastery.component.html`

Reply:
Done